package.json

```diff
@@ -20,8 +20,8 @@
 		"lint": "eslint src tests --ext ts --fix",
 		"format": "prettier --write \"{src,tests}/**/*.ts\"",
 		"docs": "typedoc",
-		"test": "jest",
-		"test:watch": "jest --watch",
+		"test": "vitest run",
+		"test:watch": "vitest",
```

Comment:
Does this do watching too?

Reply:
`vitest run` is no watch
`vitest` is watch

That said, I also recommend https://marketplace.visualstudio.com/items?itemName=ZixuanChen.vitest-explorer